roles/install/meta/main.yml

```diff
@@ -5,14 +5,10 @@ galaxy_info:
     company: Worteks
     issue_tracker_url: https://github.com/worteks/ansible-lemonldapng/issues
     license: MIT
-    min_ansible_version: 2.4
+    min_ansible_version: "2.12"
```

Comment:
Could you please explain why 2.4 is not high enough ?

Reply:
i think 2.4 is EOL since a long time, we should probably set the `min_ansible_version` to  ansible-core `2.16`